Casks/scalar-azrepos.rb

```diff
@@ -12,7 +12,7 @@
   conflicts_with cask: 'scalar'
 
   depends_on cask: 'microsoft-git'
-  depends_on cask: 'git-credential-manager-core'
+  depends_on cask: 'homebrew/cask/git-credential-manager'
```

Comment:
Why does this one have the `homebrew/cask` prefix, but `microsoft-git` doesn't?

Reply:
Ah this is what I get for not reading your links - `git-credential-manager` is part of the core Homebrew tap (rather than the Microsoft-specific one), as mentioned [here](https://github.com/microsoft/homebrew-git/pull/75#issuecomment-1681550289).